sdk/communication/azure-communication-identity/azure/communication/identity/_shared/utils.py

```diff
@@ -65,10 +65,16 @@ def create_access_token(token):
     try:
         padded_base64_payload = base64.b64decode(parts[1] + "==").decode('ascii')
         payload = json.loads(padded_base64_payload)
-        return AccessToken(token, datetime.fromtimestamp(payload['exp']).replace(tzinfo=TZ_UTC))
+        return AccessToken(token,
+            _convert_expires_on_datetime_to_utc_int(datetime.fromtimestamp(payload['exp']).replace(tzinfo=TZ_UTC)))
```

Comment:
What was this extra conversion for? Looks like we're going `str` -> `datetime` -> `int`?
Is there a reason we do the `int` conversion here, rather than when this `AccessToken` is consumed in the pipeline? Returning an int here makes it non-human-readable if a user is expected to interact with it.